.github/workflows/build.yml

```diff
@@ -51,8 +50,18 @@ jobs:
       run: |
         npm config set "//npm.pkg.github.com/:_authToken" ${{ secrets.GITHUB_TOKEN }}
         npm install
-    - name: npm build
-      run: npm run build
+      # build for home video
+    - name: npm build home video
```

Comment:
this approach doesn't layer clinic and video like I would expect.  Did you look into loading .env's based on the matrix ahead of one generic `npm run build`?